Automattic/woocommerce-payments
# Add confirmation request when submitting dispute evidence form

Fixes #382

#### Changes proposed in this Pull Request

* Request confirmation when submitting dispute evidence form.

#### Testing instructions

* Create a dispute which requires evidence form submission.
* Go to Payments > Disputes and click on arrow link near the `Needs response` in the status column.
* Click `Submit Evidence` button. Standard browser confirmation should appear:
    * Click `Cancel` to cancel the submission.
    * Click `Ok` to confirm the submission.

![382_demo](https://user-images.githubusercontent.com/3139099/74033542-72b4ea00-49c7-11ea-9c49-98aa9d3f7211.gif)


-------------------
